src/plugins/plot/MctTicks.vue

```diff
@@ -30,8 +30,8 @@
         class="gl-plot-tick-wrapper"
     >
         <div
-            v-for="tick in ticks"
-            :key="tick.value"
+            v-for="(tick, i) in ticks"
+            :key="i"
```

Comment:
We should not use indices as keys. Can you come up with a prefix to go with this?

Reply:
https://v2.vuejs.org/v2/guide/list.html?redirect=true#Maintaining-State

Reply:
Yep yep, I know about that doc. It says:

> It is recommended to provide a key attribute with v-for whenever possible, unless the iterated DOM content is simple, or you are intentionally relying on the default behavior for performance gains.

Basically putting the explicit `:key="index"` is equivalent to the default behavior (for performance gains), but I chose to explicitly write it out to make it clear.

Reply:
OK. I do see console errors when this is done, so we'll see.

Reply:
(They need to be unique only withing the `v-for`, not the whole document.)

Reply:
as per our Slack chat we changed to string-prefixed keys because they need to be unique per subtree (sibling loops conflict) so making them always unique ensures that refactoring doesn't cause a key conflict.